templates/go.mod.tpl

```diff
@@ -3,11 +3,15 @@ module github.com/onosproject/config-models/models/{{ .Name }}-{{ .Version }}
 go 1.16
 
 require (
+	github.com/antchfx/xpath v1.2.0
 	github.com/atomix/atomix-go-framework v0.10.1 // indirect
+	github.com/onosproject/config-models v0.9.1
 	github.com/onosproject/onos-api/go v0.8.14
 	github.com/onosproject/onos-lib-go v0.8.1
 	github.com/openconfig/gnmi v0.0.0-20210914185457-51254b657b7d
 	github.com/openconfig/goyang v0.3.1
 	github.com/openconfig/ygot v0.12.5
+	github.com/stretchr/testify v1.7.0
 	google.golang.org/grpc v1.41.0
+	gotest.tools v2.2.0+incompatible
 )
```

Comment:
I think you can add a `replace github.com/onosproject/config-models ../..` here because it's always from the parent folder

Reply:
@tomikazi to be more precise it is:
```
replace github.com/onosproject/config-models => ../..
```
here and in all of the `go.mod`s of the sub-modules